docs/en_US/Nnictl.md

```diff
@@ -370,6 +371,26 @@ Debug mode will disable version check function in Trialkeeper.
     nnictl experiment list
     ```
 
+* __nnictl experiment delete__
+
+  * Description
+
+    Delete the data generated by nni experiment.
```

Comment:
Change it like below,

Delete one or all experiments, it includes log, result, environment information and cache. It uses to delete useless experiment result, or save disk space.

Reply:
Fixed.